### PR TITLE
Release notes for `1.38.x - Release`

### LINKED ISSUE #22575: Android Release Notes for `1.38.x - Release`
### Release Notes:

* [Android] Add "Show all pending contributions" view #17038
* [Android] Add Gemini user wallet support #17408
* Add a way to show pending transactions while inside wallet #19837
* Replace network selector #20471
* Change Monthly Contributions to be Monthly Tips #20564
* Set custom nonce for transactions #20887
* Switching network doesn't update the default value #21136
* Add a way to reset local transaction / nonce information (android) #21457
* Show Approve/Reject dialog for unapproved transactions from Asset Details Screen/Activity #21458
* Customize Nonce in transaction panel: Android #21459
* Remove refcode from P3A reports #21460
* Approve/Reject buttons on details tab hovers over the data #21605
* [De-AMP] Follow up issues for Android #21643
* Continue executing all cosmetic scriptlets even if one fails #21932
* Add isBraveWallet property to window.ethereum #21949
* Fix unblocked cookie consent dialogs on some websites when the Cookie List is enabled #22032
* App crash when opening swap page for custom watchlist assets #22164
* Amount you entered for asset in Swap will keep changing market price and result in error #22308
* Upgrade from Chromium 101.0.4951.34 to Chromium 101.0.4951.41 #22431

### PR DESCRIPTION
**Pull request by kjozwiak**
Resolves #22575

## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A
  - [ ] https://github.com/brave/brave-browser/wiki/Brave-Wallet

## Test Plan:

N/A

### CODE REVIEW

File: CHANGELOG_ANDROID.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## [1.38.109](https://github.com/brave/brave-browser/releases/tag/v1.38.109)
+
+ - Added Gemini two-way Brave Rewards wallet. ([#17408](https://github.com/brave/brave-browser/issues/17408))
+ - Added the ability to redirect an AMP page to its canonical non-AMP version. ([#21643](https://github.com/brave/brave-browser/issues/21643))
+ - Added the ability to copy Brave Wallet transaction hash from "Transactions" on the "Portfolio" page. ([#19886](https://github.com/brave/brave-browser/issues/19886))
```
Comment (SergeyZhukovsky) [inline CHANGELOG_ANDROID.md:7]: that one is desktop related

File: CHANGELOG_ANDROID.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+ - Added the ability to set custom nonce value for Brave Wallet transactions. ([#20887](https://github.com/brave/brave-browser/issues/20887))
+ - Added the ability to customize the nonce for unapproved Brave Wallet transactions via the transactions confirmation screen. ([#21459](https://github.com/brave/brave-browser/issues/21459))
+ - Added "Clear wallet transaction and nonce information" setting for Brave Wallet under brave://settings/wallet. ([#21457](https://github.com/brave/brave-browser/issues/21457))
```
Comment (SergeyZhukovsky) [inline CHANGELOG_ANDROID.md:10]: that one is on Android but we don't have `brave://settings/wallet` it's just inside Settings menu

File: CHANGELOG_ANDROID.md (hunk 1 above)
Comment (kjozwiak) [reply CHANGELOG_ANDROID.md:7]: Nice catch, not sure how it was added as it doesn't include `OS/Android`. Removing 👍

File: CHANGELOG_ANDROID.md (hunk 2 above)
Comment (kjozwiak) [reply CHANGELOG_ANDROID.md:10]: Fixed 👍

**Review by SergeyZhukovsky — APPROVED** [review]
++

**Pull request merged**